csharp/ql/test/query-tests/Security Features/CWE-020/ExternalAPIsUsedWithUntrustedData.expected

```diff
@@ -1,2 +1,3 @@
+| System.Collections.Specialized.NameValueCollection.get_Item(string) [qualifier] | 1 | 1 |
```

Comment:
Interesting; we should model this (like we model dictionaries). (Not on this PR.)